package.json

```
@@ -1,6 +1,93 @@
 {
-  "private": true,
-  "workspaces": [
-    "packages/*"
-  ]
+  "name": "@yobgob/too-many-hooks",
+  "version": "0.0.1",
+  "description": "Collection of typed React hooks",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yobgob/too-many-hooks"
+  },
+  "keywords": [
+    "react",
+    "hooks",
+    "typescript"
+  ],
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/too-many-hooks.umd.cjs",
+  "module": "./dist/too-many-hooks.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/too-many-hooks.js",
+      "require": "./dist/too-many-hooks.umd.cjs"
+    }
+  },
+  "scripts": {
+    "build": "tsc && vite build",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
```

Now that the projects are consolidated, should we run the `build` script as part of these scripts so we always get the latest code updates?

```suggestion
    "storybook": "bun run build && storybook dev -p 6006",
    "build-storybook": "bun run build && storybook build",
```